content/docs/replication/migrating-volumes.md

```diff
@@ -0,0 +1,145 @@
+- Original volume is from the one of currently supported CSI drivers (see Support Matrix)
+- Migrated sidecar is installed alongside with the driver, you can enable it in your `myvalues.yaml` file
+```yaml
+migration:
+  enabled: true
+```
+
+## Support Matrix
+| Migration Type | PowerMax | PowerStore | PowerScale | PowerFlex | Unity | 
+| - | - | - | - | - | - | 
+| NON_REPL_TO_REPL | Yes | No | No | No | No |
+| REPL_TO_NON_REPL | Yes | No | No | No | No |
+
+
+## Basic Usage
+
+To trigger migration procedure, you need to patch existing PersistentVolume with migration annotation (by default `migration.storage.dell.com/migrate-to`) and in value of said annotation specify StorageClass name you want to migrate to. 
+
+For example, if we have PV named `test-pv` already provisioned and we want to migrate it to replicated storage class named `powermax-replication` we can run:
+
+```shell
+kubectl patch pv test-pv -p '{"metadata": {"annotations":{"migration.storage.dell.com/migrate-to":"powermax-replication"}}}'
+```
+
+Patching PV resource will trigger migration sidecar that will call `VolumeMigrate` call from the CSI driver. After migration is finished new PersistentVolume will be created in cluster with name of original PV plus `-to-<sc-name>` appended to it. 
+
+So, in case of our example, we will see the following when running `kubectl get pv`: 
+```shell
+NAME                                   CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS      CLAIM                       STORAGECLASS                REASON   AGE
+test-pv                                1Gi        RWO            Retain           Bound       default/test-pvc            powermax                             5m
+test-pv-to-powermax-replication        1Gi        RWO            Retain           Available                               powermax-replication                 10s
+
+```
+
+When Volume Migration is finished source PV will be updated with EVENT that denotes that. 
+
+Newly created PV (`test-pv-to-powermax-replication` in our example) is available for consumption via static provisioning by any PVC that will request it.
+
+
+## Namespace Considerations For Replication
+
+Replication Groups in CSM Replication can be made namespaced, meaning that one SC will generate one Replication Group per namespace, this is also important when migrating volumes from/to replcation storage class.
+
+When just setting one annotation `migration.storage.dell.com/migrate-to` migrated volume is assumed to be used in same namespace as original PV and it's PVC and in case of being migrated to replication enabled storage class will be inserted in namespaced Replication Group inside PVC namespace. 
+
+However, you can define in which namespace migrated volume must be used after migration by setting `migration.storage.dell.com/namespace`, you can use same annotation in scenario where you only have statically provisioned PV, don't have it bound to any PVC and want to migrate it to another storage class
```

Comment:
Change comma after `migration.storage.dell.com/namespace` to a period. 

Change "you can use same annotation in scenario where you only have statically provisioned PV, don't have it bound to any PVC and want to migrate it to another storage class" to read "You can use the same annotation in a scenario where you only have a statically provisioned PV, and you don't have it bound to any PVC, and you want to migrate it to another storage class"

Reply:
addressed